scope.md

```diff
@@ -19,3 +21,20 @@ hyper-specific methods for one type of data to general, do-it-all packages
 (e.g. matplotlib). pyOpenSci accepts packages that are somewhere in between the 
 two. If you're interested in submitting your data visualization package, please 
 open a pre-submission inquiry on first.
+
+## Python package technical scope
+
+pyOpenSci is still defining the technical scope criteria for package review.  
+Your package **may not be in technical scope** for us to review at this time if
+fits any of the out-of-technical-scope criteria listed below. 
+
+- Package is an out of sync fork of another package repository. 
+Sometimes we understand that a package maintainer may need to step down.  In that case, we strongly suggest that the original package owner, transfer the package to a new organization that would allow transfer of ownership of the package rather than 
```

Comment:
```suggestion
Sometimes we understand that a package maintainer may need to step down. In that case, we strongly suggest that the original package owner transfer the package to a new organization that would allow the transfer of ownership of the package rather than 
```